codegen/smithy-aws-go-codegen/src/main/java/software/amazon/smithy/aws/go/codegen/RestJsonProtocolGenerator.java

```diff
@@ -174,14 +176,53 @@ private void writeShapeToJsonArray(
                 generateSimpleShapeToJsonValue(model, writer, memberShape, operand, (w, s) -> w.write("av.$L", s));
             }
         });
-	}
+    }
 
     @Override
     public void generateProtocolTests(GenerationContext context) {
+        Set<HttpProtocolUnitTestGenerator.ConfigValue> configValues = new TreeSet<>();
+        configValues.addAll(SetUtils.of(
+                HttpProtocolUnitTestGenerator.ConfigValue.builder()
+                        .name(AddAwsConfigFields.REGION_CONFIG_NAME)
```

Comment:
`AwsConfigFields` are the client options correct?

Reply:
The AWS specific options yes